js/app/demo/app.jsx

```diff
@@ -110,10 +110,12 @@ define(["react", "jsx!editor", "jsx!messages", "jsx!fixedCode", "jsx!configurati
             }
             window.location.hash = Unicode.encodeToBase64(serializedState);
         },
-        enableFixMode: function() {
+        enableFixMode: function(event) {
+            event.preventDefault();
```

Comment:
I'm not sure this is the ideal fix. I considered the following alternatives:

* Changing the tabs to use `<a>` tags without a `href` attribute: this is apparently not very good for accessibility, and it generated some linting problems from `eslint-plugin-jsx-a11y`.
* Changing the tabs to use `<button>` tags instead of `<a>` tags: This might be a better solution, but I had some trouble getting the styling to look the same when the tags are `<button>`s.
* Setting the `href` attribute for the tabs to the current URL: This would probably have worked but it doesn't seem any better than the `event.preventDefault` solution.

I decided to go with `event.preventDefault` since it seems like the simplest option.